### DISCUSSION TITLE
Can I distribute applications that are not written in .NET languages?

### DISCUSSION THREAD

**Discussion by sparkles-laurel** [chocolatey/choco#2254] [Q&A] [answered]
Hello,
I would like to learn if I could distribute applications that are not written in .NET languages with Chocolatey.

**Comment by vexx32** [comment, marked as answer]
Yep, there's really no restriction on what you can and can't distribute as a chocolatey package. It's stored as a nupkg, but that's used more or less as just a convenient wrapper and isn't done the same strict way that (for example) nuget.org distributes packages for .NET applications.
